settings.gradle.kts

```diff
@@ -81,3 +85,68 @@ include(":modules:services:ui")
 include(":modules:services:utils")
 include(":modules:services:views")
 include(":modules:services:sharedtest")
+
+val developerProperties = loadPropertiesFromFile(File("${rootDir.path}/developer.properties"))
+val secretsFile = File("${rootDir.path}/secret.properties")
+val secretProperties = loadPropertiesFromFile(secretsFile)
+val USE_REMOTE_BUILD_CACHE_LOCALLY = "use_remote_build_cache_locally"
+
+buildCache {
+    local.directory = file("${System.getProperty("user.home")}/build-cache-test")
```

Comment:
- [ ] TODO: Remove this line before merge

Reply:
FYI: I usually add a merge instructions section on the PR description itself so that I don't forget a specific instruction before merging a PR, not sure if that helps more... 🤷